modules/client/src/client-common/constants.ts

```diff
@@ -62,7 +62,7 @@ export const LIVE_CONTRACTS: { [K in SupportedNetworks]: NetworkDeployment } = {
     tokenVotingSetup: activeContractsList.mumbai.TokenVotingSetup,
     ensRegistry: activeContractsList.mumbai.ENSRegistry,
   },
-  matic: {
+  maticmum: {
```

Comment:
This was OK as it was, now `maticmum` is duplicated

Reply:
you are right. changed back